packages/react-components/src/templates/__tests__/PersonalInfoModal.test.tsx

```diff
@@ -139,19 +164,22 @@ it('disables the form elements while submitting', async () => {
     new Promise<void>((resolve) => {
       resolveSubmit = resolve;
     });
-  const { getByText } = render(
-    <PersonalInfoModal {...props} onSave={handleSave} />,
+  const { getByText, getAllByRole } = render(
+    <PersonalInfoModal
+      {...props}
+      countrySuggestions={['United States', 'Mexico']}
+      country="Mexico"
+      onSave={handleSave}
+    />,
     { wrapper: StaticRouter },
   );
 
   userEvent.click(getByText(/save/i));
-
-  const form = getByText(/save/i).closest('form')!;
-  expect(form.elements.length).toBeGreaterThan(1);
-  [...form.elements].forEach((element) => expect(element).toBeDisabled());
+  getAllByRole('textbox').forEach((field) => expect(field).toBeDisabled());
```

Comment:
This should not have been changed. While saving all form elements save & cancel are disabled during the save not just text boxes